web/client/cypress/e2e/validator.cy.js

```diff
@@ -0,0 +1,93 @@
+/// <reference types="cypress" />
+
+context('GTFS Validator - Core Workflow', () => {
+  let numReportCalls = 0;
+
+  beforeEach(() => {
+    cy.visit('https://gtfs-validator.mobilitydata.org/')
+  });
+
+  it('Validate GTFS .zip file', () => {
+    const url = 'https://developers.google.com/static/transit/gtfs/examples/sample-feed.zip';
+    const jobId = '8f6be6fb-1fee-41f8-b401-b2b4b552e177';
+
+    // Setup intercept aliases
+    cy.intercept(
+        'POST',
+        'https://gtfs-validator-web-mbzoxaljzq-ue.a.run.app/create-job',
```

Comment:
Can we use a wildcard here like `https://*/create-job`? This hostname looks unstable and there probably aren't other requests to `/create-job` on this page that we'd need to disambiguate from with the hostname